amazon-sqs/5.8/modules/ROOT/pages/amazon-sqs-connector-reference.adoc

```diff
@@ -352,32 +412,32 @@ Deletes the message queue represented by this object. The connector can delete a
 |Type |String
```

Comment:
Same applies here, String returned by this operation contains ID of the request, shouldn't we mention it here?